ports/somnokid/Somnokid.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+
+get_controls
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+# Variables
+GAMEDIR="/$directory/ports/somnokid"
+TOOLDIR="$GAMEDIR/tools"
+
+# Exports
+export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/libs:$LD_LIBRARY_PATH"
```

Comment:
As discussed with Cebion in [PR 818](https://github.com/PortsMaster/PortMaster-New/pull/818#discussion_r1808166816) it's better and easier in the long run.

Don't forget to also rename the `libs` folder to `libs.aarch64`

```suggestion
export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/libs.${DEVICE_ARCH}:$LD_LIBRARY_PATH"
```